src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
```

Comment:
block cant have external indents

https://en.bem.info/methodology/css/#external-geometry-and-positioning

Reply:
Fixed